smithy-cli/src/it/java/software/amazon/smithy/cli/InitCommandTest.java

```diff
@@ -253,6 +261,43 @@ public void executesInitSuccessQuiet() {
         });
     }
 
+    @Test
+    public void badTemplatePathFailureExpected() {
+        IntegUtils.withProject(PROJECT_NAME, templatesDir -> {
+            setupTemplatesDirectory(templatesDir);
+
+            IntegUtils.withTempDir("badTemplatePath", dir -> {
+                RunResult result = IntegUtils.run(
+                        dir, ListUtils.of("init", "-t", "bad-template-path", "-u", templatesDir.toString()));
+                System.out.println(result.getOutput());
+                assertThat(Files.exists(Paths.get(dir.toString(), "bad-template-path")), is(false));
+                assertThat(result.getExitCode(), is(1));
+                assertThat(result.getOutput(),
+                        containsString("Template path `getting-started-example/does-not-exist` specified in"
+                                + " template bad-template-path is invalid."));
```

Comment:
Might be nice to use commas or quoting around names, otherwise the sentence reads funny